doc/release-notes/7373-solr-upgrade.md

```diff
@@ -0,0 +1,11 @@
+### Solr Update
+
+With this release we upgrade to the latest available stable release in the Solr 8.x branch. We recommend a fresh installation of Solr 8.8.1 (the index will be empty)
```

Comment:
Do we want to consider porting the old index and incremental index all? (what have we done for past Solr upgrades?)

Reply:
I didn't consider that. This is close to the release note we used when we went to 7.7.2